packages/documentation/src/stories/foundation/accessibility/regulation/regulation.stories.ts

```diff
@@ -2,7 +2,7 @@ import { StoryObj } from '@storybook/web-components';
 import { MetaExtended } from '@root/types';
 
 const meta: MetaExtended = {
-  id: '759a427c-7884-474e-b6b7-de32d3acf5df',
+  id: 'ea0e885a-ab89-492a-8905-afbed7ad9548',
```

Comment:
Why did you change the id here? Most of the time ids shouldn't be changes as it also changes the page URL.

Reply:
i got an error about duplicate ids, but it was coming from a different file after all.